compiler/src/lowering/mappers/expressions.rs

```diff
@@ -163,3 +165,32 @@ fn expr_tuple(context: &Context, exp: Node<fe::Expr>) -> fe::Expr {
 
     unreachable!()
 }
+
+fn expr_list(context: &Context, exp: Node<fe::Expr>) -> fe::Expr {
+    let attributes = context.get_expression(&exp).expect("missing attributes");
+
+    if let Type::Array(array) = &attributes.typ {
+        let fn_name = list_expr_generator_fn_name(array);
+
+        if let fe::Expr::List { elts } = exp.kind {
+            let args = elts
+                .into_iter()
+                .map(|list_val| fe::CallArg::Arg(list_val).into_node())
+                .collect::<Vec<_>>()
+                .into_node();
+
+            let _self: &'static str = Object::Self_.into();
+            // Turn List Expression into a function call
+            return fe::Expr::Call {
+                func: fe::Expr::Attribute {
+                    value: fe::Expr::Name(_self.to_string()).into_boxed_node(),
```

Comment:
I'm sure there's a nicer way to get the string of the `Object::Self` that would allow me to move it back inline here but I wasn't able to find it. /cc @sbillig

Reply:
I guess the best way is probably to derive `strum::ToString` instead of (or in addition to) `IntoStaticStr`, and do `fe::Expr::Name(Object::Self_.to_string())`

Reply:
haha, it's that simple, thanks!